pkgs/development/libraries/zlib/default.nix

```diff
@@ -23,23 +23,17 @@ assert splitStaticOutput -> static;
 
 stdenv.mkDerivation (rec {
   pname = "zlib";
-  version = "1.2.11";
+  version = "1.2.12";
 
   src = fetchurl {
     urls =
       [ "https://www.zlib.net/fossils/zlib-${version}.tar.gz"  # stable archive path
         "mirror://sourceforge/libpng/zlib/${version}/zlib-${version}.tar.gz"
       ];
-    sha256 = "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1";
+    sha256 = "c3e5e9fdd522222b542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1";
```

Comment:
What is the SHA256  `c3e5e9fdd522222b542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1`? It looks like the old SHA256 `c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1` with a few bits corrupted.

The actual SHA256 of zlib should be  `91844808532e5ce316b3c010929493c0244f3d37593afd6de04f71821d5136d9`.

Reply:
Ah, pushed to early. Fixed.